src/v/kafka/security/acl_store.h

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2021 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+#pragma once
+#include "kafka/security/acl.h"
+
+#include <absl/container/btree_map.h>
+#include <absl/container/flat_hash_set.h>
+
+namespace kafka {
+
+/*
+ * A collection of ACL entries.
+ */
+class entry_set {
```

Comment:
`kafka::entry_set` doesn't really say `kafka::acl_entry_set` meaning, `entry_set` in the parent namespace, could be ... anything.

Reply:
good call. fixed